components/automate-ui/angular.json

```diff
@@ -26,6 +26,9 @@
             ],
             "scripts": [
               "src/modernizr-custom.js"
+            ],
+            "allowedCommonJsDependencies": [
+              "hogan.js"
```

Comment:
When running `ng-build`, angular now throws a warning for any packages that have dependencies on CommonJS.  I searched for an `es` version of hogan.js but haven't come across one.  This addition to the angular.json suppresses that warning.